charts/nebula-crawler/templates/cron.yaml

```diff
@@ -0,0 +1,220 @@
+# CRAWL
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-crawl
+spec:
+  schedule: {{ .Values.crawler.schedule }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: nebula-crawler
+            chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+            heritage: {{ .Release.Service }}
+            release: {{ .Release.Name }}
+        spec:
+          restartPolicy: OnFailure
+          containers:
+            - name: crawl
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+              ports:
+                - containerPort: 6666
+                  name: metrics
+              command: [ "nebula" ]
+              args: [ "crawl" ]
```

Comment:
If you also want to collect neighbor information you may consider to add the `--neighbors` flag. This will, however, roughly double the data it collects on each crawl.